drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -209,7 +209,7 @@ def unit_types
       capacity = project_units.group(:unit_type_id).count
       unoccupied = project_units.unoccupied_on.group(:unit_type_id).count
 
-      object.units.map(&:unit_type).uniq.compact.map do |unit_type|
+      object.units.preload(:unit_type).map(&:unit_type).uniq.compact.map do |unit_type|
```

Comment:
Could be even simpler I think- no need to load all the units, we already have the `id`s of the UnitType records we need?
```suggestion
      Hmis::UnitType.where(id: capacity.keys).order(:id).distinct.map do |unit_type|
```
(Untested, with sorting for consistency)

Reply:
Looks good, I added a spec test to confirm no n+1